test-runner-poc/environments.js

```diff
@@ -0,0 +1,225 @@
+'use strict';
+const semver = require('semver');
+const path = require('path');
+const core = require('../lib/core');
+/**
+ * Base class for environments in projects that use the test
+ * runner
+ */
+class EnvironmentBase {
+  /**
+   * The default implementation of the environment setup
+   *
+   * @param {*} callback
+   */
+  setup(callback) {
+    callback();
+  }
+  constructor(parsedURI) {
+    this.host = parsedURI.hosts[0].host;
+    this.port = parsedURI.hosts[0].port;
+  }
+}
+const genReplsetConfig = (port, options) => {
+  return Object.assign(
+    {
+      options: {
+        bind_ip: 'localhost',
+        port: port,
+        dbpath: `${__dirname}/../db/${port}`,
+        setParameter: ['enableTestCommands=1']
+      }
+    },
+    options
+  );
+};
+function usingUnifiedTopology() {
+  return !!process.env.MONGODB_UNIFIED_TOPOLOGY;
+}
+/**
+ *
+ * @param {*} discoverResult
```

Comment:
Did this come from the old `environments.js` file? It seems like it may no longer be relevant.

Reply:
removed